README.md

```diff
@@ -44,25 +42,25 @@ Subsequent requests to the bucket will therefore skip the Lambda function entire
 
 The function will allow resizing of images, by width, height, or both dimensions. Example paths:
 
-- `/scaled/500x500/foobar.jpg`: this would generate a 500x500 version of the image `foobar.jpg`.
-- `/scaled/500x/foobar.jpg`: omitting the `height` would generate a 500px wide version of the image.
-- `/scaled/x500/foobar.jpg`: omitting the `width` would generate a 500px high version of the image.
+- `/scaled/width:500_height:500/foobar.jpg`: this would generate a 500x500 version of the image `foobar.jpg`.
+- `/scaled/width:500/foobar.jpg`: omitting the `height` would generate a 500px wide version of the image.
+- `/scaled/height:500/foobar.jpg`: omitting the `width` would generate a 500px high version of the image.
 
 ### File conversion
 
-You can double up on the extension to force a different output format.
+You can convert an image to a different format.
 
-- `/scaled/500x500/foobar.jpg.webp`: this will create a **webp** version of the file `foobar.jpg`, scaled to 500x500.
+- `/scaled/width:500_height:500_convert:webp/foobar.jpg.webp`: this will create a **webp** version of the file `foobar.jpg`, scaled to 500x500.
```

Comment:
Same here: I think it's a good idea to put some extra emphasis on the fact that webp is both an option and an extension.